sdk/metric/integrator/simple/simple.go

```diff
@@ -42,21 +43,28 @@ type (
 	}
 
 	batchMap map[batchKey]batchValue
+
+	batch struct {
+		sync.RWMutex
```

Comment:
Would be helpful to make it clear this isn't used locally to guard the following map.

```suggestion
		// Implements the locking for the `CheckpoingSet` interface.
		sync.RWMutex
```

Reply:
Done.